internal/du/azure_blob_inventory.go

```diff
@@ -0,0 +1,198 @@
+package du
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"slices"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/jmoiron/sqlx"
+	"github.com/oriser/regroup"
+	"golang.org/x/exp/maps"
+)
+
+type AzureBlobInventoryReportDuReader struct {
+	azureStorageConnectionString string
+	blobInventoryContainer       string
+}
+
+type rulesRanByDate = map[time.Time][]string
+
+const (
+	runDatePathFormat  = "2006/01/02/15-04-05"
+	maxSaneCountDuRows = 1e7
```

Comment:
zou even een comment erachter zetten met het werkelijke getal. Of iig iets in decimalen.

Reply:
en ik had extra e7 gebruikt zodat je meteen ziet dat het 7 nullen zijn ;)